### ISSUE TITLE
Qt6/Android 12+ : Initial Screen- Lottie Animations Not loading

### ISSUE THREAD

**Issue by strseb** [mozilla-mobile/mozilla-vpn-client#3156]
Start VPN - Initial screen should be showing an animation for each page. 

Just a black screen is shown

`{{Lottie failure: Failed to initialize the lottie component: ReferenceError: HybridRenderer is not defined - line: 23 qrc:/nebula/components/VPNLottieAnimation.qml: 22 }}```

My current hypothesis: 

```c++
  QJSValue obj = engine()->newObject();
  obj.setProperty("container", engine()->toScriptValue(m_container));
  obj.setProperty("renderer", "canvas");
  obj.setProperty("rendererSettings", rendererSettings);
  obj.setProperty("loop", m_loops);
  obj.setProperty("autoplay", m_autoPlay);
  obj.setProperty("animationData", jsonData);

  QJSValue animation =
      loadAnimation.callWithInstance(m_lottieInstance, QList<QJSValue>{obj});
```

This object is not getting passed on correctly as HybridRenderer is the default. However we should be passing "renderer:canvas" to the loadAnimation call :/ 

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1987)

**Comment by data-sync-user** [comment]
➤ Sebastian Streich commented:

Valentina Virlics - Do you have an android 12 device to double check if this also happens for you? (maybe just a me issue)? Thanks!

**Comment by data-sync-user** [comment]
➤ Maria Berlinger commented:

Hello,

This is verified as fixed using Mozilla VPN 2.9.0(16533260). The v8 Android build works as expected using Android 12(Google Pixel 4XL).

See the attached screen record.

!20220524-140058-720x1520.mp4|width=720,height=1520!
